docs/index.md

```diff
@@ -92,7 +92,7 @@ or as a development dependency for your project:
 $ npm install --save-dev mocha
 ```
 
-> To install Mocha v3.0.0 or newer with `npm`, you will need `npm` v2.14.2 or newer.  Additionally, to run Mocha, you will need Node.js v4 or newer.
+> To install Mocha v3.0.0 or newer with `npm`, you will need `npm` v2.14.2 or newer.  Additionally, to run Mocha, you will need Node.js v6 or newer.
 
 Mocha can also be installed via [Bower](https://bower.io) (`bower install mocha`), and is available at [cdnjs](https://cdnjs.com/libraries/mocha).
```

Comment:
We should zap the note about Bower while we're here.

Reply:
I didn't understand what you mean

Reply:
I mean that we should remove line 97, because new Mocha versions are no longer installable via Bower.

Reply:
I suppose that can be a separate PR.

Reply:
I got it. I will make another PR.